Orange/canvas/config.py

```diff
@@ -44,6 +45,11 @@ def init():
     QCoreApplication.setApplicationVersion(version)
     QSettings.setDefaultFormat(QSettings.IniFormat)
 
+    # Hack to make sure the correct plugin search path is added. (Github issue #1143)
+    for path in site.getsitepackages():
+        if path.endswith("site-packages"):
+            QCoreApplication.addLibraryPath(path + "\\PyQt4\\plugins")
```

Comment:
Does this work with POSIX paths on Mac OS X and GNU/Linux? Perhaps better to use `os.path.join()`.